perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -302,6 +302,12 @@ export default {
         return available > 0 ? available : 0;
       }
     },
+    iconClass () {
+      return !this.$vuetify.rtl && 'mr-2 ' || 'ml-2';
+    },
+    isItLeftSidedLanguage() {
+      return !this.$vuetify.rtl; 
```

Comment:
we don't need this anymore!

Reply:
308-309: We need it to get the menu direction when language change and avoid getting the menu options hidden .
306-307: We need to get the icons spacing the way UI demands